subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyUnresolvedModuleIntegrationTest.groovy

```diff
@@ -243,9 +243,10 @@ class DependencyUnresolvedModuleIntegrationTest extends AbstractHttpDependencyRe
         !downloadedLibsDir.isDirectory()
 
         where:
-        reason                          | abort | action              | outcome
-        'exceeds timeout'               | true  | 'expectGetBlocking' | 'assertDependencyMetaDataReadTimeout'
-        'returns internal server error' | false | 'expectGetBroken'   | 'assertDependencyMetaDataInternalServerError'
+        reason                          | abort | action                | outcome
+        'exceeds timeout'               | true  | 'expectGetBlocking'   | 'assertDependencyMetaDataReadTimeout'
+        'returns internal server error' | true  | 'expectGetBroken'     | 'assertDependencyMetaDataInternalServerError'
+        'returns uncritical error'      | false | 'expectGetUnofficial' | 'assertDependencyMetaDataSiteFrozenError'
```

Comment:
❌ All 5xx are critical. We should use something like 401 as an example of "uncritical".

Reply:
done